examples/tfengine/generated/team/example-prod-apps/main.tf

```diff
@@ -32,16 +32,15 @@ resource "google_compute_address" "static" {
 # Deletion lien: https://cloud.google.com/resource-manager/docs/project-liens
 # Shared VPC: https://cloud.google.com/docs/enterprise/best-practices-for-enterprise-organizations#centralize_network_control
 module "project" {
-  source  = "terraform-google-modules/project-factory/google//modules/shared_vpc"
-  version = "~> 9.2.0"
+  source  = "terraform-google-modules/project-factory/google//modules/svpc_service_project"
```

Comment:
I thought we're using root module now for shared vpc attachments to?

Reply:
I'll leave that as a separate task in the backlog as it's not blocking any immediate usage and I'm not 100% sure the root module can do exactly the same thing as the `svpc_service_project` submodule. Filed https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/issues/697